src/libraries/System.Private.CoreLib/src/System/Reflection/Emit/TypeNameBuilder.cs

```diff
@@ -0,0 +1,445 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+/*
+	This TypeNameBuilder is modified from CoreCLR's original.
+	It replaces the C++ bits of the implementation with a faithful C# port.
+	Original files:
+		src/coreclr/src/System.Private.CoreLib/src/System/Reflection/Emit/AQNBuilder.cs
```

Comment:
This change is deleting AQNBuilder.cs. Comment referring to non-existent file is not very useful.